### ISSUE TITLE
raven.js Error when signed in

### ISSUE THREAD

**Issue by JoshRosenstein** [CompuIves/codesandbox-client#894]
# 🐛 bug report

## Description of the problem
Cannot view any sandboxes post login. Errors thrown from raven.js. Only after logging in. 

## How has this issue affected you? What are you trying to accomplish?
Blank screen...
### Your Environment
Error occurs in multiple Browsers.

## Temp Solution
Got everything back to normal after request blocking raven.min.js 

## Console Error
````
raven.js:55 TypeError: Cannot read property 'name' of undefined
    at index.js:24
    at beginWork (react-dom.production.min.js:132)
    at r (react-dom.production.min.js:161)
    at i (react-dom.production.min.js:161)
    at a (react-dom.production.min.js:162)
    at w (react-dom.production.min.js:169)
    at x (react-dom.production.min.js:168)
    at batchedUpdates (react-dom.production.min.js:171)
    at ut (react-dom.production.min.js:54)
    at Xt (mobx.module.js:3206)
    at Qt (mobx.module.js:3182)
    at It (mobx.module.js:2730)
    at mobx.module.js:947
    at F (mobx.module.js:914)
    at n (mobx.module.js:902)
    at mobx-state-tree.module.js:1635
    at Q (mobx-state-tree.module.js:1545)
    at Object.<anonymous> (mobx-state-tree.module.js:1560)
    at t.value (Model.js:89)
    at t.value (Model.js:141)
    at ProviderConstructor.e.(anonymous function) [as set] (https://codesandbox.io/static/js/app.adf3bc04.js:1:138512)
    at Object.n [as function] (set.js:23)
    at e.value (FunctionTree.js:98)
    at t (executeTree.js:76)
    at executeTree.js:26
    at FunctionTree.js:115
a.(anonymous function) @ raven.js:55
s @ react-dom.production.min.js:164
a @ react-dom.production.min.js:162
w @ react-dom.production.min.js:169
x @ react-dom.production.min.js:168
batchedUpdates @ react-dom.production.min.js:171
ut @ react-dom.production.min.js:54
Xt @ mobx.module.js:3206
Qt @ mobx.module.js:3182
It @ mobx.module.js:2730
(anonymous) @ mobx.module.js:947
F @ mobx.module.js:914
n @ mobx.module.js:902
(anonymous) @ mobx-state-tree.module.js:1635
Q @ mobx-state-tree.module.js:1545
(anonymous) @ mobx-state-tree.module.js:1560
value @ Model.js:89
value @ Model.js:141
e.(anonymous function) @ State.js:34
operator.set(state`user`, props`user`) @ set.js:23
value @ FunctionTree.js:98
t @ executeTree.js:76
(anonymous) @ executeTree.js:26
(anonymous) @ FunctionTree.js:115
Promise.then (async)
value @ FunctionTree.js:112
t @ executeTree.js:76
p @ executeTree.js:14
(anonymous) @ executeTree.js:31
value @ FunctionTree.js:166
t @ executeTree.js:76
(anonymous) @ executeTree.js:26
(anonymous) @ FunctionTree.js:115
Promise.then (async)
value @ FunctionTree.js:112
t @ executeTree.js:76
p @ executeTree.js:14
(anonymous) @ executeTree.js:31
value @ FunctionTree.js:166
t @ executeTree.js:76
t @ executeTree.js:44
(anonymous) @ executeTree.js:80
a @ FunctionTree.js:314
value @ FunctionTree.js:331
value @ BaseController.js:166
run @ Module.js:36
onClick @ index.js:158
N @ react-dom.production.min.js:26
invokeGuardedCallback @ react-dom.production.min.js:25
invokeGuardedCallbackAndCatchFirstError @ react-dom.production.min.js:25
X @ react-dom.production.min.js:30
ee @ react-dom.production.min.js:32
ne @ react-dom.production.min.js:32
J @ react-dom.production.min.js:31
se @ react-dom.production.min.js:34
Yt @ react-dom.production.min.js:93
Kt @ react-dom.production.min.js:70
batchedUpdates @ react-dom.production.min.js:171
ut @ react-dom.production.min.js:54
Zt @ react-dom.production.min.js:71
d @ raven.js:418
raven.js:422 Uncaught TypeError: Cannot read property 'name' of undefined
    at index.js:24
    at beginWork (react-dom.production.min.js:132)
    at r (react-dom.production.min.js:161)
    at i (react-dom.production.min.js:161)
    at a (react-dom.production.min.js:162)
    at w (react-dom.production.min.js:169)
    at x (react-dom.production.min.js:168)
    at batchedUpdates (react-dom.production.min.js:171)
    at ut (react-dom.production.min.js:54)
    at Xt (mobx.module.js:3206)
    at Qt (mobx.module.js:3182)
    at It (mobx.module.js:2730)
    at mobx.module.js:947
    at F (mobx.module.js:914)
    at n (mobx.module.js:902)
    at mobx-state-tree.module.js:1635
    at Q (mobx-state-tree.module.js:1545)
    at Object.<anonymous> (mobx-state-tree.module.js:1560)
    at t.value (Model.js:89)
    at t.value (Model.js:141)
    at ProviderConstructor.e.(anonymous function) [as set] (https://codesandbox.io/static/js/app.adf3bc04.js:1:138512)
    at Object.n [as function] (set.js:23)
    at e.value (FunctionTree.js:98)
    at t (executeTree.js:76)
    at executeTree.js:26
    at FunctionTree.js:115
(anonymous) @ index.js:24
beginWork @ react-dom.production.min.js:132
r @ react-dom.production.min.js:161
i @ react-dom.production.min.js:161
a @ react-dom.production.min.js:162
w @ react-dom.production.min.js:169
x @ react-dom.production.min.js:168
batchedUpdates @ react-dom.production.min.js:171
ut @ react-dom.production.min.js:54
Xt @ mobx.module.js:3206
Qt @ mobx.module.js:3182
It @ mobx.module.js:2730
(anonymous) @ mobx.module.js:947
F @ mobx.module.js:914
n @ mobx.module.js:902
(anonymous) @ mobx-state-tree.module.js:1635
Q @ mobx-state-tree.module.js:1545
(anonymous) @ mobx-state-tree.module.js:1560
value @ Model.js:89
value @ Model.js:141
e.(anonymous function) @ State.js:34
operator.set(state`user`, props`user`) @ set.js:23
value @ FunctionTree.js:98
t @ executeTree.js:76
(anonymous) @ executeTree.js:26
(anonymous) @ FunctionTree.js:115
setTimeout (async)
(anonymous) @ raven.js:1059
(anonymous) @ BaseController.js:223
(anonymous) @ FunctionTree.js:310
value @ FunctionTree.js:102
t @ executeTree.js:76
(anonymous) @ executeTree.js:26
(anonymous) @ FunctionTree.js:115
Promise.then (async)
value @ FunctionTree.js:112
t @ executeTree.js:76
p @ executeTree.js:14
(anonymous) @ executeTree.js:31
value @ FunctionTree.js:166
t @ executeTree.js:76
(anonymous) @ executeTree.js:26
(anonymous) @ FunctionTree.js:115
Promise.then (async)
value @ FunctionTree.js:112
t @ executeTree.js:76
p @ executeTree.js:14
(anonymous) @ executeTree.js:31
value @ FunctionTree.js:166
t @ executeTree.js:76
t @ executeTree.js:44
(anonymous) @ executeTree.js:80
a @ FunctionTree.js:314
value @ FunctionTree.js:331
value @ BaseController.js:166
run @ Module.js:36
onClick @ index.js:158
N @ react-dom.production.min.js:26
invokeGuardedCallback @ react-dom.production.min.js:25
invokeGuardedCallbackAndCatchFirstError @ react-dom.production.min.js:25
X @ react-dom.production.min.js:30
ee @ react-dom.production.min.js:32
ne @ react-dom.production.min.js:32
J @ react-dom.production.min.js:31
se @ react-dom.production.min.js:34
Yt @ react-dom.production.min.js:93
Kt @ react-dom.production.min.js:70
batchedUpdates @ react-dom.production.min.js:171
ut @ react-dom.production.min.js:54
Zt @ react-dom.production.min.js:71
d @ raven.js:418

**Comment by CompuIves** [comment]
Hey! Thanks for the report, I think this one is a duplicate of #893. I just deployed a fix, could you try a refresh? Don't hesitate to re-open if you still have the issue.
